pilot/pkg/networking/core/v1alpha3/fake.go

```diff
@@ -140,6 +140,10 @@ func NewConfigGenTest(t test.Failer, opts TestOptions) *ConfigGenTest {
 	env.ServiceDiscovery = serviceDiscovery
 	env.IstioConfigStore = model.MakeIstioStore(configController)
 	env.NetworksWatcher = opts.NetworksWatcher
+	env.JwtKeyResolver = model.NewJwksResolver(model.JwtPubKeyEvictionDuration, model.JwtPubKeyRefreshInterval, model.JwtPubKeyRetryInterval)
+	t.Cleanup(func() {
```

Comment:
nit: `t.Cleanup(env.JwtKeyResolver.Close)` can be used

Reply:
Updated, thanks!